CMakeLists.txt

```diff
@@ -1009,6 +1014,7 @@ set(PRELEX_SCOOUT ${CMAKE_CURRENT_BINARY_DIR}/csound_prslex.c)
 #    COMMAND ${FLEX_EXECUTABLE} ARGS -B -t ${LEX_SCOSRC} > ${LEX_SCOOUT}
 #    )
 
+file(MAKE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/Engine)
```

Comment:
Should you really have to manaully create these folders? If it's output here, it should make it at the same time. There aren't any other manually created folders in the binary dir

Reply:
I thought so to, was working locally for me without this but CI failed without manually making the folders. Maybe something to do with permissions?